docs/reference/ingest/processors/inference.asciidoc

```diff
@@ -149,4 +149,33 @@ The mapping field name for {feat-imp} is compounded as follows:
 `ml.<inference.target_field>`.`<inference.tag>`.`feature_importance`
 
 If `inference.tag` is not provided in the processor definition, it is not part 
-of the field path. The `<inference.target_field>` defaults to `inference`.
\ No newline at end of file
+of the field path. The `<inference.target_field>` defaults to `inference`.
+
+For example, you provide a tag `foo` in the definition as you can see below:
+
+[source,js]
+--------------------------------------------------
+{
+  "tag": "foo",
+  ...
+}
+--------------------------------------------------
+// NOTCONSOLE
+
+
+The `{feat-imp}` value is written to the `ml.inference.foo.feature_importance` 
+field.
+
+You can also specify a target field as follows:
+
+[source,js]
+--------------------------------------------------
+{
+  "tag": "foo",
+  "target_field": "my_field"
+}
+--------------------------------------------------
+// NOTCONSOLE
+
+In this case, `{feat-imp}` is exposed in the 
+`ml.my_field.foo.feature_importance` field.
```

Comment:
it is not `ml.my_field`. Its just `my_field.foo.feature_importance`

Reply:
@benwtrent Thanks. I also updated the schema accordingly. I.e.: `<ml.inference.target_field>`.`<inference.tag>`.`feature_importance` instead of 
`ml.<inference.target_field>`.`<inference.tag>`.`feature_importance`